index.bs

```diff
@@ -1347,6 +1379,11 @@ The second controls the maximum number of distinct [=sites=] across all [=attrib
 for [=attribution sources=] with a given ([=attribution source/source site=], [=attribution source/reporting origin=] [=site=])
 per [=destination rate-limit window=].
 
+<dfn>Max destinations per reporting site per day</dfn> is an integer controls
```

Comment:
Should this be "per source reporting site"? Otherwise this seems to be across source sites by naming.

Reply:
Updated to "per source reporting site", thanks.